docs/installation.md

```diff
@@ -4,7 +4,7 @@ In this document, we will discuss the overall steps on how
 to download and set up the PHGv2 package.
 
 ## Quick start
-* Run on Linux (_Windows not currently tested_)
+* Run on Unix-style operating system (_Windows not currently tested_)
```

Comment:
I sort of think "Unix-based" sounds better. But up to you

Reply:
Updated. Thanks.